docs/en/drivers/ios-xcuitest.md

```diff
@@ -129,10 +129,17 @@ following desired capabilities:
 such as: biometrics, geolocation setting and window focussing.
 
 ```
-# Get the Facebook Tap.
-brew tap facebook/fb
```

Comment:
is the formula removed from brew?

Reply:
No, but it only provides 0.4.0 and HEAD for now.
The 0.4.0 is old. HEAD fails to build, today.

I was able to checkout out v1.0.6 tag from the repo and got `/bin/fbsimctl`. But not sure the command will remain so far, sine commands built by `build.sh` decreased than `v1.0.0` tag.

Thus, so far, getting the repo, checkout `v1.0.0` and built binaries are safer way.